docs/Deployment/ManualDeploymentExtended.md

```diff
@@ -1300,4 +1300,8 @@ Updating could break things. Be careful on a live system.
 ~$ sudo systemctl start rtl
 ```
 
-## 🏁 The End
+##  The End
+
+### Questions
+
+Join the [community chat](https://chat.btcpayserver.org/) on Mattermost by downloading [Mattermost app](https://mattermost.com/download/), or on [Telegram](https://t.me/btcpayserver) in case you need further help or help or want to hang around with like-minded people.
```

Comment:
Why is this needed on every page you document? 😄

Reply:
I've mainly touched bigger topic pages. 
If you look at Dashboard's PR, you will find I did not put it there. 
As that is a straight forward page, this extended manual Deployment might raise questions by user. :)